resources/lang/id/tournament.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+return [
+    'index' => [
+        'header' => [
+            'subtitle' => 'Daftar turnamen yang aktif dan diakui secara resmi.',
+            'title' => 'Turnamen Komunitas',
+        ],
+        'none_running' => 'Tidak ada turnamen yang berlangsung saat ini, silakan periksa lagi nanti!',
+        'registration_period' => 'Pendaftaran: :start sampai :end',
+    ],
+    'show' => [
+        'button' => [
+            'register' => 'Daftarkan saya!',
+            'cancel' => 'Batalkan Pendaftaran',
+        ],
+        'entered' => 'Anda terdaftar di turnamen ini.<br><br>Perhatikan hal ini bukan berarti Anda telah dimasukkan ke dalam tim.<br><br>Instruksi lebih lanjut akan dikirimkan melalui email saat turnamen akan dimulai, jadi tolong pastikan alamat email akun osu! Anda valid!',
```

Comment:
Here's a wording suggestion that I believe would work better in regards to the tournament registration's context : 

+        'entered' => 'Anda telah berhasil mendaftarkan diri Anda pada turnamen ini. <br><br>Mohon diperhatikan bahwa hal ini bukan berarti Anda telah secara otomatis telah diikutsertakan ke dalam salah satu tim yang bertanding. <br><br>Instruksi lebih lanjut akan dikirimkan melalui email saat turnamen akan dimulai, jadi mohon pastikan alamat email akun osu! Anda valid!',